sotodlib/site_pipeline/update_det_match.py

```diff
@@ -63,6 +63,16 @@ class UpdateDetMatchesConfig:
     write_relpath: bool
         If True, will use the relative path to the h5 file (relative to the db
         path) when writing to the manifestdb
+    solution_type: str
+        Type of solutions to use. Must be one of ['kaiwen_handmade',
+        'resonator_set'].  If 'kaiwen_handmade', will use the handmade solutions
+        from Kaiwen pulled from the wafer_map file in the site-pipeline-configs.
+        If `resonator_set`, must also specify the ``resonator_set_dir`` to pull
+        solutions from.
+    resonator_set_dir: Optional[str]
+        If ``solution_type`` is 'resonator_set', this must be specified and
+        contain the path to the resonator-set solutions. This directory must
+        have a res-set npy file for each stream_id that is expected in the matching.
```

Comment:
Can you connect this explicitly to the name of the class in the code that would write out the npy file?  I.e. "as written by ResonatorSet.save()" or whatever.  Also, filename pattern is required here?